README.md

```diff
@@ -9,6 +9,7 @@
 [![Test Coverage](https://img.shields.io/codecov/c/github/18F/openFEC/master.svg)](https://codecov.io/github/18F/openFEC)
 [![Code Climate](https://img.shields.io/codeclimate/github/18F/openFEC.svg)](https://codeclimate.com/github/18F/openFEC)
 [![Dependencies](https://img.shields.io/gemnasium/18F/openFEC.svg)](https://gemnasium.com/18F/openFEC)
+[![Code Issues](https://www.quantifiedcode.com/api/v1/project/5e91408cecf44ce8bb8d9f1b904bdd99/badge.svg)](https://www.quantifiedcode.com/app/project/5e91408cecf44ce8bb8d9f1b904bdd99)
```

Comment:
@NoahKunin: It looks like this points to your fork of the repo, which won't stay current with upstream unless you're syncing manually. Also, do we prefer quantified code to code climate for python apps?